examples/vision/semisupervised_simclr.py

```diff
@@ -0,0 +1,743 @@
+"""
+Title: Semi-supervised image classification using contrastive pretraining with SimCLR
+Author: [András Béres](www.linkedin.com/in/andras-beres-789190210)
+Date created: 2021/04/24
+Last modified: 2021/04/24
+Description: Using SimCLR for contrastive pretraining for semi-supervised image classification.
+"""
+"""
+## Introduction
+
+### Semi-supervised learning
```

Comment:
In this section it would be useful to reference this example https://keras.io/examples/vision/simsiam/ and explain what is different & what is similar here